python/openlocationcode.py

```diff
@@ -379,7 +379,7 @@ def clipLatitude(latitude):
  have different precisions due to the grid method having fewer columns than
  rows.
 """
-def computeLatitutePrecision(codeLength):
+def computeLatitudePrecision(codeLength):
```

Comment:
This might be an API change, but you really should make it...

Reply:
I'll accept it - it's not a part of the publicly declared API.